GOVERNANCE.md

```diff
@@ -0,0 +1,138 @@
+
+## Adding maintainers
+
+Maintainers are first and foremost contributors that have shown they are
+committed to the long term success of a project. Contributors wanting to become
+maintainers are expected to be deeply involved in contributing code, pull
+request review, and triage of issues in the project for more than three months.
+
+Just contributing does not make you a maintainer, it is about building trust
+with the current maintainers of the project and being a person that they can
+depend on and trust to make decisions in the best interest of the project.
+
+Periodically, the existing maintainers curate a list of contributors that have
+shown regular activity on the project over the prior months. From this list,
+maintainer candidates are selected and proposed on the project mailing list.
+
+After a candidate has been announced on the project mailing list, the
+existing maintainers are given fourteen business days to discuss the candidate,
+raise objections and cast their vote. Votes may take place on the mailing list
+or via pull request comment. Candidates must be approved by at least 66% of the
+current maintainers by adding their vote on the mailing list. The reviewer role
+has the same process but only requires 33% of current maintainers. Only
+maintainers of the repository that the candidate is proposed for are allowed to
+vote.
+
+If a candidate is approved, a maintainer will contact the candidate to invite
+the candidate to open a pull request that adds the contributor to the
+MAINTAINERS file. The voting process may take place inside a pull request if a
+maintainer has already discussed the candidacy with the candidate and a
+maintainer is willing to be a sponsor by opening the pull request. The candidate
+becomes a maintainer once the pull request is merged.
+
+## Subprojects
+
+Serverless Workflow subprojects all culminate in officially supported and maintained releases
+of the specification. 
+All subprojects must adhere to [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md)
+as well as this governance document.
+
+### Adding core subprojects
+
+New subprojects can request to be added to the Serverless Workflow GitHub
+organization by submitting a GitHub issue in the specification repository.
+
+The existing maintainers are given fourteen business days to discuss the new
+project, raise objections and cast their vote. Projects must be approved by at
+least 66% of the current maintainers.
+
+If a project is approved, a maintainer will add the project to the Serverless Workflow
+GitHub organization, and make an announcement on a public Slack channel.
+
+## Stepping down policy
+
+Life priorities, interests, and passions can change. If you're a maintainer but
+feel you must remove yourself from the list, inform other maintainers that you
+intend to step down, and if possible, help find someone to pick up your work.
+At the very least, ensure your work can be continued where you left off.
+
```

Comment:
Some projects have an emeritus list

Reply:
i this no matter what the governance is or not we are all community contributors, and noone will/can be removed from that title :) maybe one day when we have a big community we could add. wdyt